src/components/atoms/Text/Text.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import styles from './Text.scss';
+
+const Text = ({ children, className, tag, theme, ...props }) => {
+    const TagName = tag;
+    const classNames = cx({
+        [className]: className,
+        [styles.headline]: theme === 'headline',
```

Comment:
If these are mutually exclusive visual states, meaning 2 cannot be applied at once, we should only apply the theme once. Something like: `const themeClass = styles[theme];`

Reply:
Another good call. Updated: https://github.com/MadeInHaus/react-redux-webpack-starter/pull/34/commits/457f670fc40d827710fda0125ca53a20210f5a9e